she_codes_news/news/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from django.forms import ModelForm
+from .models import NewsStory
+
+
+class StoryForm(ModelForm):
+    class Meta:
+        model = NewsStory
+        fields = ["title", "category", "pub_date", "content", "story_image"]
+        widgets = {
+            "title": forms.TextInput(attrs={"class":"form-control"}),
+            "category": forms.TextInput(attrs={"class":"form-control"}),
+            "pub_date": forms.DateInput(format=("%m%d%Y"),
+            attrs={"class": "form-control", "placeholder": "Select a date", "type": "date", "title": "Publication Date"}),
+            "content": forms.Textarea(attrs={"class":"form-control"})
```

Comment:
putting the same class on every field is a lot of repeating yourself, you could just put a class on the form itself